etc/README.md

```diff
@@ -0,0 +1,8 @@
+# Config
+
+Test configuration to configure Wedge 40 with the following port config:
+
+ - Port 1: QSFP+ -> SFP+ converter with a 1GE SFP Copper module in it
+ - Port 2-16: 4x10G QSFP+
+
+Status: not working, switch on other end does not show any packets ingressing
```

Comment:
Can you comment on the "status: not working".  Is this still the case?  Because it's this type of problem that I would expect to see for the bring up process not working in the way I had thought it would.

Reply:
As I said in the other issue it's the case of an unfortunate wording. This particular issue our attempts at linking a 1G link using 1G SFP modules.